.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": [
+    "node-services",
+    "prettier"
+  ],
+  "plugins": [
+    "prettier"
+  ],
+  "rules": {
+    "prettier/prettier": "error"
+  },
+  "parserOptions": {
+    "ecmaVersion": 2016,
+    "sourceType": "module"
+  },
+  "env": {
+    "es6": true,
+    "node": true
```

Comment:
Most definitely. I haven't dealt with that before so I'd like to discuss some options about how to do it if you don't mind.

I've created https://phabricator.wikimedia.org/T172683 to discuss and not forget